src/ui/details.py

```
@@ -0,0 +1,63 @@
+HTML = str
+
+
+def render_citation(citation: Citation) -> HTML:
+    return f"""
+        <blockquote style="margin-top: 1em; border-left: 2px solid #ccc; padding-left: 1em;">
+            <p><i>"{citation.text}"</i></p>
+            <footer>— {citation.author}</footer>
+        </blockquote>
+    """
+
+from typing import Literal
```

Ruff (0.12.2)

Module level import not at top of file

(E402)